packages/cli/src/workflows/workflow.service.ee.ts

```diff
@@ -249,14 +249,14 @@ export class EnterpriseWorkflowService {
 		]);
 		NotFoundError.isDefinedAndNotNull(
 			workflow,
-			`Could not find workflow with the id "${workflowId}". Make sure you have the permission to delete it.`,
+			`Could not find workflow with the id "${workflowId}". Make sure you have the permission to move it.`,
```

Comment:
Unrelated fix. The error message is outdated. Initially I required the delete scope, but I created a new scope since.